@navikt/core/react/src/accordion/AccordionHeader.tsx

```diff
@@ -19,13 +20,22 @@ const AccordionHeader = forwardRef<HTMLButtonElement, AccordionHeaderProps>(
     const itemContext = useContext(AccordionItemContext);
     const accordionContext = useContext(AccordionContext);
 
+    const themeContext = UNSAFE_useAkselTheme(false);
+
     if (itemContext === null) {
       console.error(
         "<Accordion.Header> has to be used within an <Accordion.Item>, which in turn must be within an <Accordion>",
       );
       return null;
     }
 
+    let headingSize = accordionContext?.headingSize ?? "small";
+
+    if (themeContext) {
+      /* Fallback to "medium" if any other sizes are used */
+      headingSize = accordionContext?.size === "small" ? "xsmall" : "small";
```

Comment:
Is the comment correct? Seems to me that it falls back to "small"?

Reply:
Clarified the comment a little since i was referencing the "size"-prop in Accordion
```
      /* Fallback to "medium" Accordion-size if any other sizes are used */

```